seed/data_importer/tasks.py

```diff
@@ -1768,6 +1811,12 @@ def pair_new_states(merged_property_views, merged_taxlot_views, sub_progress_key
         # if pv.state.lot_number and ";" in pv.state.lot_number:
         #     pdb.set_trace()
 
+        # Check if 'pv' is an array, then
+        # use the first element, otherwise use 'pv' as is
+        if isinstance(pv, list):
+            if len(pv) > 0:
+                pv = pv[0]
+
```

Comment:
Can you explain why this was added?

Reply:
This is no longer necessary. Thanks for reminding me to take this out.